packages/client/components/SpotlightModal.tsx

```diff
@@ -0,0 +1,214 @@
+  justifyContent: 'center',
+  overflow: 'hidden',
+  width: isDesktop ? '80vw' : '90vw'
+}))
+
+const SelectedReflection = styled('div')({
+  alignItems: 'flex-start',
+  background: PALETTE.SLATE_100,
+  borderRadius: '8px 8px 0px 0px',
+  display: 'flex',
+  flexWrap: 'wrap',
+  height: '33.3%',
+  justifyContent: 'center',
+  padding: 16,
+  position: 'relative',
+  width: '100%'
+})
+
+const SimilarReflectionGroups = styled('div')({
+  display: 'flex',
+  justifyContent: 'center',
+  height: '66.6%',
+  padding: 16
+})
+
+const Title = styled('div')({
+  color: PALETTE.SLATE_800,
+  fontSize: 16,
+  fontWeight: 600,
+  textAlign: 'center'
+})
+
+const TopRow = styled('div')({
+  width: `calc(100% - 48px)`,
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center'
+})
+
+const Content = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  height: '100%',
+  position: 'absolute',
+  top: 0
+})
+
+const SearchInput = styled('input')({
+  appearance: 'none',
+  border: `1px solid ${PALETTE.SKY_500}`,
+  borderRadius: 4,
+  boxShadow: `0 0 1px 1px ${PALETTE.SKY_300}`,
+  color: PALETTE.SLATE_700,
+  display: 'block',
+  fontSize: 14,
+  lineHeight: '24px',
+  outline: 'none',
+  padding: '6px 0 6px 39px',
+  width: '100%',
+  '::placeholder': {
+    color: PALETTE.SLATE_600
+  }
+})
+
+const SearchItem = styled(MenuItemLabel)({
+  overflow: 'visible',
+  padding: 0,
+  position: 'absolute',
+  bottom: -16,
+  width: ElementWidth.REFLECTION_CARD
+})
+
+const StyledMenuItemIcon = styled(MenuItemComponentAvatar)({
+  position: 'absolute',
+  left: 8,
+  top: 8
+})
+
+const SearchIcon = styled(Icon)({
+  color: PALETTE.SLATE_600,
+  fontSize: ICON_SIZE.MD24
+})
+
+const StyledCloseButton = styled(PlainButton)({
+  height: 24,
+  position: 'absolute',
+  right: 16
+})
+
+const CloseIcon = styled(Icon)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  fontSize: ICON_SIZE.MD24,
+  '&:hover,:focus': {
+    color: PALETTE.SLATE_800
+  }
+})
+
+interface Props {
+  closeSpotlight: () => void
+  meeting: SpotlightModal_meeting
+  reflection: SpotlightModal_reflection
+}
+
+const SpotlightModal = (props: Props) => {
+  const {closeSpotlight, meeting, reflection} = props
+  const isDesktop = useBreakpoint(Breakpoint.NEW_MEETING_SELECTOR)
+  const handleKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
+    if (e.key === 'Escape') closeSpotlight()
+  }
+  const reflectionGroupsCount = 0
+  return (
+    <ModalContainer onKeyDown={handleKeyDown} isDesktop={isDesktop}>
+      <SelectedReflection>
+        <TopRow>
+          <Title>Find cards with similar reflections</Title>
+          <StyledCloseButton onClick={closeSpotlight}>
+            <CloseIcon>close</CloseIcon>
+          </StyledCloseButton>
+        </TopRow>
+        <Content>
+          <DraggableReflectionCard
+            inSpotlight
+            isReadOnly
+            staticIdx={0}
+            reflection={reflection as any}
+            meeting={meeting as any}
```

Comment:
I've been having problems getting the right types for the meeting and reflection. It looks like we may have had a similar issue in [ExpandedReflection](https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/client/components/RetroReflectPhase/ExpandedReflection.tsx#L15). Ideally, I'd like to just query the bare minimum fields we need in the file, the reflection & meeting ids, but I'm having difficulty as TypeScript complains about missing refs. I think this is because there's a circular reference: the reflection card contains the spotlight modal which contains the reflection card